app/services/ipam/external_ipam.rb

```diff
@@ -0,0 +1,33 @@
+module IPAM
+  class ExternalIpam < Base
+    delegate :external_ipam_proxy, :to => :subnet
+
+    def suggest_ip
+      if self.mac.nil?
+        errors.add(:mac, "You must specify a MAC address before selecting the External IPAM subnet")
+        return nil
+      end
+
+      cidr = @subnet.network + '/' + @subnet.cidr.to_s
```

Comment:
There's already a helper for this:
```suggestion
      cidr = @subnet.network_address
```